arm/Microsoft.Compute/virtualMachines/.parameters/windows.min.parameters.json

```diff
@@ -2,11 +2,14 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-vm-win-02"
```

Comment:
How about
```suggestion
            "value": "<<namePrefix>>-vm-winmin-001"
```

Reply:
VM name has a limit of 15 chars for Windows